.github/project_workflows/deploy_app_store.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         fetch-depth: 0
 
     - name: Run SwiftLint
-      uses: norio-nomura/action-swiftlint@3.1.0
+      uses: norio-nomura/action-swiftlint@3.2.1
```

Comment:
Improvement: Might need some investigation if this syntax works so we don't have to come up date this all the time https://stackoverflow.com/a/73259281/3736373.
```
      uses: norio-nomura/action-swiftlint@v3
```

Reply:
I updated to  `uses: norio-nomura/action-swiftlint@master` due to this github only has master branch 👍